core/math/random_pcg.h

```diff
@@ -126,10 +126,18 @@ class RandomPCG {
 	}
 
 	_FORCE_INLINE_ double randfn(double p_mean, double p_deviation) {
-		return p_mean + p_deviation * (cos(Math_TAU * randd()) * sqrt(-2.0 * log(randd()))); // Box-Muller transform
+		double temp = randd();
+		if (temp < CMP_EPSILON2) {
+			temp += CMP_EPSILON2; // To prevent generating of INF value in log function, resulting to return NaN value from this function.
```

Comment:
`CMP_EPSILON2` is for squared values, not relevant here:
```
#define CMP_EPSILON 0.00001
#define CMP_EPSILON2 (CMP_EPSILON * CMP_EPSILON)
```

Reply:
I thought it for double type cuz 0.00001 * 0.00001 = 0.000000001